### PR TITLE
BasicAESClearKey sample updated to use Track2 SDK

### PR DESCRIPTION
**Pull request by xpouyat**
## Purpose
- BasicAESClearkey sample updated to use Track2 SDK
- Code refactoring
- SDK updates

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: Account/CreateAccount/Program.cs [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-using Azure.Identity;
 using Azure;
 using Azure.Core;
+using Azure.Identity;
 using Azure.ResourceManager;
-using Azure.ResourceManager.Resources;
 using Azure.ResourceManager.Media;
 using Azure.ResourceManager.Media.Models;
+using Azure.ResourceManager.Resources;
+using Common_Utils;
```
Comment (jonpayne) [inline Account/CreateAccount/Program.cs:10]: Why is `Common_Utils` needed? (where possible, I'm trying to make the samples standalone so users don't need to look in multiple files to understand how to use the SDK).

File: Assets/Program.cs [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Globalization;
 using Azure;
 using Azure.Identity;
 using Azure.ResourceManager;
 using Azure.ResourceManager.Media;
+using System;
```
Comment (jonpayne) [inline Assets/Program.cs:8]: Normally our coding convention is to order `System.` usings before other usings (but I'm not too concerned either way).

**Review by jonpayne — APPROVED** [review]

File: Account/CreateAccount/Program.cs (hunk 1 above)
Comment (xpouyat) [reply Account/CreateAccount/Program.cs:10]: I did not change the code, just did a reordering.
Common_Utils is used for loading the settings.

// Load configuration using the Common Utilities library
ConfigWrapper config = Common_Utils.DotEnv.LoadEnvOrAppSettings();

We need to define our policy for the settings for all the samples.
We can get rid of common_Utils but it means will will duplicate the ConfigWarapper class and LoadEnvOrAppSettings function in each sample.

File: Assets/Program.cs (hunk 2 above)
Comment (xpouyat) [reply Assets/Program.cs:8]: Visual Studio default code clean up tool is ordering alphabetically.

File: Account/CreateAccount/Program.cs (hunk 1 above)
Comment (johndeu) [reply Account/CreateAccount/Program.cs:10]: Fine for now, but agree we should try to make them more standalone.  Reason is that @IngridAtMicrosoft  is trying to get these to show up as a single page in the docs with all the code needed in one file, like the TypeScript and Python samples.  Less complex spaghetti code is good for the docs.

File: Assets/Program.cs (hunk 2 above)
Comment (johndeu) [reply Assets/Program.cs:8]: Do whatever the public tooling is doing for VS or VS Code.

File: Common_Utils/Common_Utils.csproj [hunk 3]
```diff
@@ -6,14 +6,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.8.0" />
+    <PackageReference Include="Azure.Identity" Version="1.8.1" />
     <PackageReference Include="Azure.ResourceManager.Media" Version="1.0.0" />
     <PackageReference Include="Azure.Storage.Blobs" Version="12.14.1" />
     <PackageReference Include="Microsoft.Azure.Management.Media" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="7.0.0" />
```
Comment (johndeu) [inline Common_Utils/Common_Utils.csproj:13]: What is this Extensions.Configuration.Abstractions library?

**Review by johndeu — APPROVED** [review]

**Pull request merged**